app/services/BackgroundTaskService.js

```diff
@@ -0,0 +1,123 @@
+import BackgroundFetch from 'react-native-background-fetch';
+import PushNotification from 'react-native-push-notification';
+import { IntersectSet } from '../helpers/Intersect';
+import { GetStoreData, SetStoreData } from '../helpers/General';
+import { isPlatformiOS } from './../Util';
+
+const INTERSECT_INTERVAL = 30 / 60; // In Minutes
```

Comment:
I think to make sense a create like 'constants.js' file for all constants of the app.

Reply:
that's a good point, makes it much more consistent. I think a good idea would be to port all of them in a separate PR

Reply:
Yeah, I will create a new issue for it and will resolve it in a separate PR. Thanks!